src/BannerViewManager.js

```diff
@@ -0,0 +1,23 @@
+import { PropTypes } from 'react';
+import { requireNativeComponent, View } from 'react-native';
+
+
+const BANNER_HEIGHT_50 = { width: -1, height: 50 };
+const BANNER_HEIGHT_90 = { width: -1, height: 90 };
+const RECTANGLE_HEIGHT_250 = { width: -1, height: 250 };
+
+const banner = {
+  name: 'BannerView',
+  propTypes: {
+    placementId: PropTypes.string,
+    size: PropTypes.oneOf([BANNER_HEIGHT_50, BANNER_HEIGHT_90, RECTANGLE_HEIGHT_250]),
+    ...View.propTypes
+  },
+};
+
+const BannerView = requireNativeComponent('CTKBannerView', banner);
+BannerView.BANNER_HEIGHT_50 = BANNER_HEIGHT_50;
+BannerView.BANNER_HEIGHT_90 = BANNER_HEIGHT_90;
+BannerView.RECTANGLE_HEIGHT_250 = RECTANGLE_HEIGHT_250;
```

Comment:
If there are only 3 types of Banner we can keep `BannerView` internally and export 3 components with applied `size` property, @satya164 @ferrannp @grabbou what do you think?